vmware-esxi/x86_64/7.md

```diff
@@ -0,0 +1,12 @@
+# Changelog for vmware_esxi_7_0
```

Comment:
Add a date to the title, so we can add a new one next time we update the image. Also, assume this file will have new content at the top (rather than appended).